src/utils/minFee.ts

```diff
@@ -0,0 +1,19 @@
+import { BigNumber } from 'bignumber.js'
+
+export const DEFAULT_GAS_PRICE = 40e9 // 40 Gwei
+export const ETH_PRICE_IN_OWL = 240 * 1000
```

Comment:
That's not really accurate, is it?

Reply:
super not accurate, especially the last few days 📈 
as a placeholder for now